framework/packages/abstract-adapter/src/endpoints/ibc_callback.rs

```diff
@@ -11,22 +11,9 @@ impl<Error: ContractError, CustomInitMsg, CustomExecMsg, CustomQueryMsg, Receive
     fn ibc_client(&self, deps: Deps) -> Result<Addr, Self::Error> {
         let vc_query_result = self
             .abstract_registry(deps)?
-            .query_module(
-                ModuleInfo::from_id_latest(IBC_CLIENT).map_err(|err| {
-                    let err: AbstractSdkError = err.into();
-                    err
-                })?,
-                &deps.querier,
-            )
-            .map_err(|err| {
-                let err: AbstractError = err.into();
-                let err: AbstractSdkError = err.into();
-                err
-            })?;
+            .query_module(ModuleInfo::from_id_latest(IBC_CLIENT)?, &deps.querier)
```

Comment:
This was discussed before, because for adapters, there is no associated account (remeber, the ibc client is calling this endpoint).
For the apps, we need to make sure that the ibc-client is installed so using the query from the account directly

Reply:
Yes, we decided to use the specific IBC contract version when resolving so we can update the implementation in the future.